util/topgen/templates/BUILD.tpl

```diff
@@ -14,10 +14,16 @@ load(
 )
 
 # IP Integration Tests
+% for min in range(0, len(irq_peripheral_names), 10):
```

Comment:
It would be nice if this was a skylark loop not a template loop, it would make the generated BUILD file a little more readable.

Reply:
I changed the test to be generated from starlark loop, but I keep the test suite a template loop so it'll be easy to spot if we (ever) change the number of peripheral to the point that number of tests changed and we have to update DV files accordingly.